packages/auth/package.json

```diff
@@ -49,7 +49,7 @@
     "branches": 80
   },
   "devDependencies": {
-    "@aerogear/core": "2.1.0-rc2",
+    "@aerogear/core": "2.1.0",
```

Comment:
should this be in `dependencies` rather than `devDependencies`?

Reply:
I put core into peerDependencies for runtime and devDependencies for typescript builds.
We can put core to be in dependencies for each package as form of simplification. 
WDYT?

Reply:
It actually made no difference now but having peerDependencies on those will always make sure that there is this single core there. Core then will come from app module that requires that. However this is more complex idea so maybe let's just put core into dependencies and call it a day. 
Doing this now.

Reply:
+1

Reply:
Ah.. our scripts for version bump putting core into dependencies that is why it landed there. Yes another good reason to not change this.